### PR TITLE
refactor(sanity): remove commented publishat input

### PR DESCRIPTION
**Pull request by RitaDias**
### Description

Made the input use the generic date inputs.
This is temporary since the design for the dialog will likely change this is useful for being able to create releases with published dates

### What to review

Code changes

### Testing

Can you create a release with a publishAt date (in the future)?
Does it appear properly?

### CODE REVIEW

File: packages/sanity/src/core/form/inputs/DateInputs/CommonDateTimeInput.tsx
Comment (RitaDias) [inline packages/sanity/src/core/form/inputs/DateInputs/CommonDateTimeInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/form/inputs/DateInputs/DateInput.tsx
Comment (RitaDias) [inline packages/sanity/src/core/form/inputs/DateInputs/DateInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/form/inputs/DateInputs/DateTimeInput.tsx
Comment (RitaDias) [inline packages/sanity/src/core/form/inputs/DateInputs/DateTimeInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/form/inputs/DateInputs/utils.ts
Comment (RitaDias) [inline packages/sanity/src/core/form/inputs/DateInputs/utils.ts:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/scheduledPublishing/components/dateInputs/base/DateTimeInput.tsx
Comment (RitaDias) [inline packages/sanity/src/core/scheduledPublishing/components/dateInputs/base/DateTimeInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/scheduledPublishing/components/dateInputs/base/calendar/YearInput.tsx
Comment (RitaDias) [inline packages/sanity/src/core/scheduledPublishing/components/dateInputs/base/calendar/YearInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/core/tasks/components/form/fields/DateEditFormField.tsx
Comment (RitaDias) [inline packages/sanity/src/core/tasks/components/form/fields/DateEditFormField.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/DatePicker.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/DatePicker.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/DateTimeInput.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/DateTimeInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/LazyTextInput.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/LazyTextInput.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/calendar/Calendar.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/calendar/Calendar.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/calendar/CalendarDay.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/calendar/CalendarDay.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

File: packages/sanity/src/ui-components/inputs/DateInputs/calendar/CalendarMonth.tsx
Comment (RitaDias) [inline packages/sanity/src/ui-components/inputs/DateInputs/calendar/CalendarMonth.tsx:1]: will be removed once https://github.com/sanity-io/sanity/pull/7410 is merged and corel branch is rebased

**Review by bjoerge — APPROVED** [review]

**Pull request merged**